### PR TITLE
Add unit tests to generated dialogs

### LINKED ISSUE #4783: Unit tests are required for generated dialogs
## Describe the bug
Currently, for generated dialogs, there is only one end-to-end test for it. That is not sufficient. Each dialog event kind for the generated dialogs (e.g,, OnAssignEntity, OnChooseEntity, OnChooseProperty) requires the unit test.

### PR DESCRIPTION
**Pull request by coldplaying42**
Fixes #4783 

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: tests/tests.uischema [hunk 1]
```diff
@@ -37,99 +37,138 @@
...
       "helpLink": "https://aka.ms/bfc-ask-for-user-input",
-      "label": "Prompt with multi-choice",
+      "label": "Prompt for a number",
       "properties": {
         "property": {
           "intellisenseScopes": [
             "variable-scopes"
           ]
         }
       },
```
Comment (chrimc62) [inline tests/tests.uischema:92]: Don't think you should check-in this file.

**Review by chrimc62 — APPROVED** [review]
:shipit:

**Comment by mrivera-ms** [comment]
@coldplaying42 , this PR needs a better title, description and a linked issue. Please add those elements before we can merge.

**Comment by coldplaying42** [comment]
@mrivera-ms I have added the linked issue.

**Pull request merged**